src/services/useDaoMetadata.ts

```diff
@@ -0,0 +1,43 @@
+import fetchJson from '@/utils/fetchJson'
```

Comment:
Also, I wouldnt recommend putting hooks inside of services. 

Hooks belong to the React domain. services have no relation to hooks. Service can be called by hooks, other methods, etc. 

Please create a tree like this. 

- hooks
- - useDaoMetatdata
- services
- - daoMetadata
- - - index.ts // this will call and export the API
- - - types.ts // this will contain the interfaces.